### ISSUE TITLE
[Request] Static placeholder image for webcam to prevent jumps on control tab

### ISSUE THREAD

**Issue by CptanPanic** [foosel/OctoPrint#478]
It may be because I am running octoprint on a raspberry pi, but everytime I switch to control tab, it takes a few seconds to load the camera image.  Since there is no size listed for the img tag, the controls start up at top of page, and then after the image loads, then they get shifted down towards the bottom.  Is it possible to make this size static?

**Comment by foosel** [comment]
TODO:

  * [ ] set fixed aspect ratio for webcam (4:3, 16:9 will become letter boxed, no way to determine image size prior to loading)
  * [ ] provide placeholder image while stream is still loading, to prevent controls from jumping
  * [ ] if stream URL produces error, offer checkbox to disable stream loading for session (see #609)

**Comment by foosel** [comment]
This is now implemented on the `maintenance` branch, soon `devel` and will be part of the upcoming 1.3.3 release.

In the end I did it slightly differently than the above TODO list suggests.

We now have a webcam placeholder at aspect ratio of 16:9 or 4:3 (configurable) - well, or their portrait orientation equivalent if the webcam stream is rotated, a "Webcam stream loading..." message, a nice little error message if the loading went wrong, and no jumping controls anymore.
